fix inconsistencies (#29)
in-toto/docs: in-toto-spec.md

a client will install.

To avoid ambiguity, we will refer to any files in the final product that

> in-toto does not use to verify supply chain integrity as "target files". Target

Reviewer: I believe American English _always_ puts the period within the quotes. @jhdalek55 could I trouble you for an opinion? :)

Reviewer 2: Yes. Inside the quotes. Another way we differ from British English.

Reviewer: Thank you!

Author: Oh ok, but then we should be at least consistent through out the specification right? Because I observed both cases sometimes "foobar". and "foobar."

Reviewer: I think those would belong in this PR. Thoughts?

Author: Mhh I thought about this for a while, don't we want to emphasize the word with the double quotes? I agree that in American English you would write: "Hello, I am fine."

But we want to emphasize specific words, we don't have written speech here.
Let's have a look on this sentence here:

`An example date-time string is 1985-10-21T01:21:00Z.`

When we emphasize the date-time string we will get the following sentence with "American Quotation":

`An example date-time string is "1985-10-21T01:21:00Z."`

For me as an European this looks odd and looks like the "dot" at the end of the date-time string would be part of the date-time string. What do you guys think? Maybe it's just my European brain.

I had a look on RFCs and they do the "European Quotation". For example (https://tools.ietf.org/html/rfc2822):

```
   For brevity, this document sometimes
   refers to this range of characters as simply "US-ASCII characters".
```

Reviewer 3: As a fellow European I support the period outside the quotes. :)

}
```

> Where, ROLE is a dictionary whose `"\_type"` field describes the metadata type (as

Reviewer 3: Now that we mark this up as verbatim text, the escape character `\` becomes visible, so we should remove it.

Author: Oops, you are right. I didn't see this. I will make a PR ready that removes the backslash

}
```

The NAME string will be used to identify this step within the supply chain.
NAME values are unique and so they MUST not repeat in different step
descriptions.

> The `"threshold"` field must contain an integer value indicating how many pieces

Reviewer 4: ```suggestion
The `"threshold"` field must contain a positive integer indicating how many pieces
```
